websites/P/Podurama/metadata.json

```
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "358402930191106049",
+		"name": "ion606"
+	},
+	"service": "Podurama",
+	"description": {
+		"en": "Google Podcast presence, but with the podcast artwork"
```

The official description may be better
```suggestion
		"en": "Best free podcast player to listen to free podcasts, add custom rss feeds or upload your private files on mobile and desktop devices. Discover over 2M podcasts and millions of episodes."
```